clients/src/main/java/org/apache/kafka/common/metrics/Sensor.java

```diff
@@ -37,16 +38,20 @@
     private final List<KafkaMetric> metrics;
     private final MetricConfig config;
     private final Time time;
+    private volatile long lastRecordTime;
+    private final long expireInactiveSensorTimeMillis;
```

Comment:
Changed to ms. Having said that, I found numerous instances in code where we refer to values as "xxMillis" and the associated config as "Ms". Perhaps we should standardize this in the future and fix all existing instances.